src/sentry/runner/commands/cleanup.py

```diff
@@ -182,6 +183,8 @@ def is_filtered(model):
             (models.GroupRuleStatus, "date_added", None),
             (models.RuleFireHistory, "date_added", None),
             (monitor_models.MonitorCheckIn, "date_added", None),
+            (metrics_indexer_models.StringIndexer, "last_seen", None),
+            (metrics_indexer_models.PerfStringIndexer, "last_seen", None),
```

Comment:
Ahh, it looks like we conditionally use postgres as part of the indexer based on `SENTRY_METRICS_INDEXER`.

So I think that the way you should add these is to add them to `EXTRA_BULK_QUERY_DELETES` programatically when we instantiate https://github.com/getsentry/sentry/blob/0c225ed01bd03f92ff4925cfa019b551fab9684f/src/sentry/sentry_metrics/indexer/postgres/postgres_v2.py#L212-L214

That way, if we configure a different indexer we'll stop attempting to clean this up.

Reply:
Thanks, by programmatically I'm assuming you mean something like
```python
def __init__(self) -> None:
        super().__init__(CachingIndexer(indexer_cache, PGStringIndexerV2()))
        cleanup.EXTRA_BULK_QUERY_DELETES.extend(
            [
                (StringIndexer, "last_seen", None),
                (PerfStringIndexer, "last_seen", None),
            ]
        )
```